### PR TITLE
Mar/ipld1.0

### PR DESCRIPTION
**Pull request by mrodriguez3313**
- added objectives goals to objectives page
- added goals to intro page
- finished objectives & ipld1.0 
   - introduced background terms for each section
   - added new images

### CODE REVIEW

File: content/en/curriculum/ipld/content-addressing-cids/index.md [hunk 1]
```diff
@@ -10,17 +10,29 @@ category: lecture
...
+objectives:
+  show: true
+  goals:
+  - "1.1"
+  subgoals:
+  - 1.11
+  - 1.12
+  - 1.13
 ---
+## Background Information
```
Comment (mrodriguez3313) [inline content/en/curriculum/ipld/content-addressing-cids/index.md:22]: Note: this page contains edits related to ipld1.1

File: content/en/curriculum/ipld/introduction/index.md [hunk 2]
```diff
@@ -55,14 +53,21 @@ At its most fundamental, **IPFS** is a collection of:
...
-### Tutorials
+### Prequel to IPLD
+
+The CID that is returned to you when you add a file to IPFS is the entry point to these graph structures. **Graphs** are data structures to organize all the blocks of data pertaining to the respective file. This organization is handled by **"links"**; links help create a logical order of blocks for IPLD to address every subsequent block of data. There will be more on graphs in a later lesson. 
+
+![Picture of a graph](tree_graph.png)
 
+{{% level type="[deep]" %}}
+### Tutorials
+s
```
Comment (enoldev) [inline content/en/curriculum/ipld/introduction/index.md:70]:
```suggestion

```

File: content/en/curriculum/ipld/introduction/index.md [hunk 3]
```diff
@@ -55,14 +53,21 @@ At its most fundamental, **IPFS** is a collection of:
...
 For those who are newer to the world of Filecoin, Web3, and storage verification, check out the [Protoschool tutorials](https://proto.school/course/ipld). Tutorials you should complete include:
 
 * [Merkle DAGs: Structuring Data for the Distributed Web](https://proto.school/merkle-dags)
```
Comment (enoldev) [inline content/en/curriculum/ipld/introduction/index.md:73]: I think we can link this tutorials in their corresponding section:
"Merkle DAGs" in the Merkle DAGs section
"Anatomy of a CID" in the Content Addressing & CIDs section
etc

If we include them here, we're asking people to take those tutorials before they've reached the corresponding curriculum section

File: content/en/curriculum/ipld/introduction/index.md [hunk 4]
```diff
@@ -76,6 +81,8 @@ For those who are newer to the world of Filecoin, Web3, and storage verification
 
 {{< youtube nLgxTYoF-K4 >}}
```
Comment (enoldev) [inline content/en/curriculum/ipld/introduction/index.md:82]: Not sure where this tutorials should go. I think you are working on an IPLD tutorial that replaces this one, right?

**Review by enoldev — APPROVED** [review]
Good job! Just some minor comments

File: content/en/curriculum/ipld/introduction/index.md (hunk 3 above)
Comment (mrodriguez3313) [reply content/en/curriculum/ipld/introduction/index.md:72]: Makes sense

File: content/en/curriculum/ipld/introduction/index.md (hunk 4 above)
Comment (mrodriguez3313) [reply content/en/curriculum/ipld/introduction/index.md:81]: true, Im thinking with the way we are breaking it up. the different parts (part 1,2,3) could go in different sections of the curriculum. For now I think I could move it to the pathing section.

**Pull request merged**